index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Build a Tribute Page</title>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main">
+       <h1 id="title">Dr. Norman Borlaug</h1>
+       <figure id="img-div">
+         <img id="image" src="img/img.jpeg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field"/>
+         <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+       </figure>
+       <section id="tribute-info">
+        <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia entry</a>
+       </section>
+    </main>
```

Comment:
- Great Job using Github's inline comment in highlighting lines of code that you are suggesting changes on. 💪🏽 

- I'd like to kindly suggest that it is important to state reasons WHY a change needs to be implemented. For example, how about we re write your comment to sound like this 👉🏽 ("How about putting renaming the main tag from on lines 12 and 21 to section, this will help to . . . . . . . . . . . The whole of this should be a section tag" )

- I think it will be great to consider adding some emoji's to your suggestions as this will make your comments sound a bit more friendly. 🤗 

What do you think about re phrasing your comments?? 🤔